src/Uno.UWP/Devices/Geolocation/Geolocator.Android.cs

```diff
@@ -62,9 +62,6 @@ public Task<Geoposition> GetGeopositionAsync()
 			}
 		}
 
-		public Task<Geoposition> GetGeopositionAsync(TimeSpan maximumAge, TimeSpan timeout)
```

Comment:
This is a breaking change, we cannot remove a method from the public APIs.

Reply:
Yes, it is breaking change - but method is unimplemented, really.
And PR#2536 with this method implemented is waiting since Jan 25, 2020 (PR created date; than changes because of e.g. #4061 merged).